src/ui/SmartSnippet/SmartSnippetCommon.ts

```diff
@@ -11,3 +14,14 @@ export const getDefaultSnippetStyle = (contentClassName: string) => `
     margin-bottom: 0;
   }
 `;
+
+export const transformSnippetLinks = (
+  renderedSnippetParent: HTMLElement,
+  alwaysOpenInNewWindow: boolean,
+  logAnalytics: (link: HTMLAnchorElement) => void
+) => {
+  Dom.nodeListToArray(renderedSnippetParent.querySelectorAll('a')).forEach((link: HTMLAnchorElement) => {
+    bindAnalyticsToLink(link, () => logAnalytics(link));
+    link.target = alwaysOpenInNewWindow ? '_blank' : '_top';
```

Comment:
By default, smart snippets are added to an `iframe`. Without any `target`, links are open in their iframe, so `_top` is necessary here.